modules/data.remote/R/download.thredds.R

```diff
@@ -20,84 +30,124 @@
 ##' @export
 ##' @author Bailey Morrison
 ##'
-download.thredds.AGB <- function(outdir = NULL, site_ids, run_parallel = FALSE, 
-                                    ncores = NULL) {
+download.thredds.data <- function(outdir = NULL, site_info, dates = c("19950201", "19961215"), 
```

Comment:
I recommend *not* providing a default value for `dates`, so that it fails immediately and clearly when users don't provide them
```suggestion
download.thredds.data <- function(outdir = NULL, site_info, dates, 
```

Reply:
And while I'm thinking about it, consider moving `outdir` to come after  later in the argument list -- it's best to arrange arguments such that everything without a default value comes before everything with a default value, even if the default is NULL.